src/operator/groupBy-support.ts

```diff
@@ -3,8 +3,8 @@ import {Subject} from '../Subject';
 import {Subscriber} from '../Subscriber';
 import {Observable} from '../Observable';
 
-export class RefCountSubscription<T> extends Subscription<T> {
-  primary: Subscription<T>;
+export class RefCountSubscription<T> extends Subscription {
```

Comment:
I think you can also remove the type parameter from `RefCountSubscription<T>`. What do you think?